### PR TITLE
update provisioning test jenkinsfile for further compatibility

### PR DESCRIPTION
**Pull request by slickwarren**
## Issue: 

 https://github.com/rancher/qa-tasks/issues/1321

### CODE REVIEW

**Review by markusewalker — COMMENTED** [review]
Overall, LGTM. Can you do a quick test for the cert rotation since you rearranged the `kubeProvisioningClient`?

**Review by markusewalker — APPROVED** [review]

File: tests/v2/validation/certrotation/cert_rotation.go [hunk 1]
```diff
@@ -337,7 +339,7 @@ func getCertificatesFromV3Node(client *rancher.Client, v3Node *management.Node)
 	return certificates, nil
 }
 
-func downloadRKE1SshKeys(client *rancher.Client, v3Node *management.Node) ([]byte, error) {
+func downloadRKE1SSHKeys(client *rancher.Client, v3Node *management.Node) ([]byte, error) {
```
Comment (susesgartner) [inline tests/v2/validation/certrotation/cert_rotation.go:342]: Would there be a way to accomplish this with V1 instead of V3 objects?

**Review by susesgartner — APPROVED** [review]
LGTM. I had one question however feel free to ignore it if its too much scope creep.

File: tests/v2/validation/certrotation/cert_rotation.go (hunk 1 above)
Comment (slickwarren) [reply tests/v2/validation/certrotation/cert_rotation.go:342]: this was just an update to the naming, not to the func. But perhaps, I can look into it. Can this be a separate issue from this PR?

**Review by Josh-Diamond — APPROVED** [review]

File: tests/v2/validation/certrotation/cert_rotation.go (hunk 1 above)
Comment (susesgartner) [reply tests/v2/validation/certrotation/cert_rotation.go:342]: Sure. No need to scope creep this PR.

**Pull request merged**